### PR TITLE
DOCS-5226 Azure DevOps Extension Edits

### PR DESCRIPTION
**Pull request by alai97**
## Context

Adds a site region shortcode in the doc page.

## Motivation

DOCS-5226

## Preview

https://docs-staging.datadoghq.com/alai97/continuous-testing-cicd-integrations-site-region-update/continuous_testing/cicd_integrations/azure_devops_extension

### CODE REVIEW

**Review by etnbrd — CHANGES_REQUESTED** [review]
Same as https://github.com/DataDog/synthetics-ci-github-action/pull/127#pullrequestreview-1382910107

File: README.md [hunk 1]
```diff
@@ -125,7 +124,7 @@ Your task can be [simple](#simple-usage) or [complex](#complex-usage).
...
 | `appKey`             | _optional_  | Your Datadog application key when using the `apiAppKeys` authentication type. This key is created by your [Datadog organization][6] and should be stored as a [secret][7].                                                                      |
 | `subdomain`          | _optional_  | The name of the custom subdomain set to access your Datadog application when using the `apiAppKeys` authentication type. If the URL used to access Datadog is `myorg.datadoghq.com`, this value needs to be set to `myorg`. **Default:** `app`. |
-| `datadogSite`        | _optional_  | The [Datadog site][11] when using the `apiAppKeys` authentication type. **Default:** `datadoghq.com`.                                                                                                                                           |
+| `datadogSite`        | _optional_  | The [Datadog site][11] when using the `apiAppKeys` authentication type. Your Datadog site is {{< region-param key="dd_site" code="true" >}}. **Default:** `datadoghq.com`.                                                                                                                                           |
```
Comment (alai97) [inline README.md:127]:
```suggestion
| `datadogSite`        | _optional_  | The [Datadog site][11] when using the `apiAppKeys` authentication type. **Default:** `datadoghq.com`.                                                                                                                                           |
```

**Review by etnbrd — CHANGES_REQUESTED** [review]

**Review by etnbrd — APPROVED** [review]
🚀

**Review by rtrieu — APPROVED** [review]
Looks good!

**Pull request merged**